src/Factory/Api/Mastodon/Status.php

```diff
@@ -289,7 +289,7 @@ public function createFromUriId(int $uriId, int $uid = 0, bool $display_quote =
 		}
 
 		$delivery_data = new FriendicaDeliveryData($item['delivery_queue_count'], $item['delivery_queue_done'], $item['delivery_queue_failed']);
-		$friendica     = new FriendicaExtension($item['title'], $item['changed'], $item['commented'], $item['edited'], $item['received'], $counts->dislikes, $delivery_data);
+		$friendica     = new FriendicaExtension($item['title'], $item['commented'], $item['edited'], $item['received'], $counts->dislikes, $delivery_data);
```

Comment:
I guess you wanted to removed `edited`, but not `changed`? :grin:

Reply:
should be fixed now